platform/security/sts/security-sts-realm/src/main/java/ddf/security/realm/sts/AbstractStsRealm.java

```diff
@@ -110,6 +113,10 @@
 
     private Boolean useKey = null;
 
+    private Cache<Element, SecurityToken> cache = CacheBuilder.newBuilder()
+            .expireAfterWrite(1, TimeUnit.MINUTES)
```

Comment:
❓ Will no session ever have a life of shorter than 1 minute (such that there might be a bug with caching them for up to a minute after creation) ?